server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java

```diff
@@ -421,6 +428,61 @@ public void testUpperLowercase() {
       assertEquals("Hello", response);
    }
 
+   @Test
+   void testBlpopAsync() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
this is a list command, should be in the lists test file

Reply:
that, or a new file for all the blocking commands testing

Reply:
added a new RespBlockingCommandsTest.java file